guides/expression_language.md

```diff
@@ -35,7 +35,7 @@ Comparing two values of _different_ data types defaults to `false`.
 
 The exception is `!=` (not equals) which defaults to `true`. This is in line with intuition.
 
-```rust
+```ts
```

Comment:
Why typescript? It seemed natural using `rust`.

Reply:
rhai uses ts in its documentation